test/run/run-e2e-tests.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+const { promisify } = require( 'util' );
+const { initApp } = require( './init_app' );
+const { initLogs } = require( './init_logs' );
+const { initTests } = require( './init_tests' );
+const { initDriver } = require( './init_driver' );
+const { checkCredentials } = require( './checks' );
+
+let app;
+let driver;
+
+const delay = promisify( setTimeout );
+
+// Handle both user-initiated (SIGINT) and normal termination.
+process.on( 'SIGINT', function() {
+    handleExit();
+    process.exit();
+} );
+
+process.on( 'exit', handleExit );
+function handleExit() {
+    if ( driver ) {
+        driver.kill();
+    }
+    if ( app ) {
+        app.kill();
+    }
+}
+
+async function run() {
+    try {
+        checkCredentials();
+
+        const timestamp = ( new Date() ).toJSON().replace( /:/g, '-' );
```

Comment:
can we change this to be some explicit format? I find it near impossible to know the intention behind it, especially if for some reason the JSON serialization changes and we end up with formats that we don't expect.

```js
const formatYMD = instant => {
  const year = instant.getFullYear();
  const month = `${ instant.getMonth() + 1 }`.padStart(2, '0');
  const day = `${ instant.getDate() }`.padStart(2, '0');

  return `${ year }-${ month }-${ date }`;
}

…

const timestamp = formatYMD( new Date() );
```

inline isn't so bad: it's still explicit but it is uglier on account of needing to cast numbers to strings

```js
const now = new Date();
const timestamp = `${now.getFullYear()}-${`${now.getMonth() + 1}`.padStart(2,'0')}-${`${now.getDate()}`.padStart(2, '0')}`
```

Reply:
Agree - this snippet of code isn't intuitive at first glance. However in this specific case, all the snippet is doing is some very minor re-formatting and a comment to clarify is sufficient (add [here](https://github.com/Automattic/wp-desktop/pull/778/files#diff-95f02e568cfd6b3a0b1830f8fd2306a1R65)):

```
// Replace `:` with `-` to format timestamp as YYYY-MM-DDTHH-MM-SS.mmmZ
```

I think a comment here is perfectly fine and creating an entire method to replicate all desired format components would add unnecessary complexity (and verbosity) to the file. If we choose to have a more customized date format in the future, we can create a separate method at that point.

Reply:
the comment is helpful enough since it shows what you wanted. my point was made though since I didn't expect `.mmmZ` 😄